### DISCUSSION TITLE
Development suggestions - Spring 2

### DISCUSSION THREAD

**Discussion by Dariusionut** [Dariusionut/user-management#100] [Ideas]
### You can write any development/ improvement suggestions here.😊

**Comment by gonciarstef** [comment]
Create a database table to store what course a user registered for and when that happened

**Comment by gonciarstef** [comment]
Restrict deletion of users only to admins

**Reply by Dariusionut** [reply]
Good one. I will take it in consideration.

**Comment by gonciarstef** [comment]
Add profile image to users.

**Comment by gonciarstef** [comment]
Implement "Remember me" functionality on Login

**Comment by CiceroNila** [comment]
To be able to select which courses you want to attend and to be able to see them on a calendar with the allocated hours and to alert you when they overlap (sa poti sa selectezi la ce cursuri vrei sa participi si sa le poti vedea pe un calendar cu orele alocate si sa te atentioneze cand se suprapun)

**Comment by CiceroNila** [comment]
To receive a warning email 24 before and 2 hours before the start of the course, which course you are going to take, the location, the start and end time and the topic. (sa primesti email de atentionare cu 24 inainte si cu 2 ore inainte de incepere curs cu ce curs urmeaza sa faci, locatia, ora de incepere si sfarsit si tematica.)

**Comment by CiceroNila** [comment]
See the history of the courses you attended, when the courses end and the conditions for entering the exam. (Sa vezi istoricul cursurilor la care ai participat, cand se termina cursurile si conditiile pentru intrare in examen)

**Comment by CiceroNila** [comment]
To be able to send requests for issuing certificates or other administrative issues. (Sa poti trimite solicitari pentru eliberare adeverinte sau alte probleme administrative.)

**Comment by CiceroNila** [comment]
to pay attention to the courses in which you have problems and cannot enter the exam. (sa te atentioneze la cursurile la care ai probleme si nu poti intra la examen)
